specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/desktopvirtualization.json

```diff
@@ -3637,181 +3966,496 @@
+              {
+                "value": "StartFailed",
+                "description": "Hostpool update start failed."
+              },
+              {
+                "value": "UpdateFailed",
+                "description": "Hostpool update update failed."
+              },
+              {
+                "value": "Cancelling",
+                "description": "Hostpool update cancelling."
+              },
+              {
+                "value": "Canceled",
+                "description": "Hostpool update canceled."
+              },
+              {
+                "value": "Completed",
+                "description": "Hostpool update completed."
               }
             ]
           },
           "type": "string",
-          "description": "The type of image session hosts use in the hostpool.",
+          "description": "State of hostpool update.",
           "x-nullable": false
         },
-        "marketPlaceInfo": {
-          "$ref": "#/definitions/MarketPlaceInfoProperties",
-          "description": "The values to uniquely identify a gallery image."
-        },
-        "storageBlobUri": {
+        "timeCreated": {
+          "format": "date-time",
           "type": "string",
-          "description": "The uri to the storage blob which contains the VHD. Image type must be StorageBlob."
+          "description": "The hostpool update create time."
         },
-        "customId": {
-          "type": "string",
-          "description": "The resource id of the custom image or shared image. Image type must be custom."
-        }
-      }
-    },
-    "MarketPlaceInfoProperties": {
-      "description": "Image configurations of HostPool.",
-      "type": "object",
-      "properties": {
-        "offer": {
+        "timeStarted": {
+          "format": "date-time",
           "type": "string",
-          "description": "The offer of the image."
+          "description": "The hostpool update start time."
         },
-        "publisher": {
+        "timeFailed": {
+          "format": "date-time",
           "type": "string",
-          "description": "The publisher of the image."
+          "description": "The hostpool update fail time."
         },
-        "sku": {
+        "timeEnded": {
+          "format": "date-time",
           "type": "string",
-          "description": "The sku of the image."
+          "description": "The hostpool update end time."
         },
-        "exactVersion": {
-          "type": "string",
-          "description": "The exact version of the image."
+        "listOfErrors": {
+          "type": "array",
+          "description": "The alerts given to customers for hostpool update.",
+          "items": {
+            "$ref": "#/definitions/HostPoolUpdateFault"
+          }
+        },
+        "progressPercentage": {
+          "type": "number",
+          "format": "float",
+          "description": "The progress percentage."
+        },
+        "sessionHostsToMigrate": {
```

Comment:
why use "Migrate" wording? should we use "sessionHostsToUpdate", "sessionHostsToUpdating", and "sessionHostsToUpdated"?

Reply:
@shailahirji they came up with the names

Reply:
ok to substitute Migrate with HostPoolUpdate

Reply:
do you mean "update", e.g. "sessionHostsToMigrate" => "sessionHostsToUpdate"?

Reply:
@datr93 please help to find all "migrate" usage (in any form, e.g. migration or migrated, etc.) to replace with "update".